core/domain/summary_services_test.py

```diff
@@ -30,9 +31,17 @@ class ExplorationDisplayableSummaries(
     ALBERT_NAME = 'albert'
     BOB_NAME = 'bob'
 
+    USER_C_NAME = 'c'
+    USER_D_NAME = 'd'
+    USER_C_EMAIL = 'c@example.com'
+    USER_D_EMAIL = 'd@example.com'
+
+    USER_C_PROFILE_PICTURE = 'c_profile_picture.png'
```

Comment:
Nit: can you remove the ".png"? This implies it would be a filename, whereas what is usually stored here is actually a data URI.